src/components/common/ExplorerButton/index.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ReactElement, ComponentType } from 'react'
+import { IconButton, SvgIcon, Tooltip } from '@mui/material'
+import LinkIcon from '@/public/images/common/link.svg'
+
+export type ExplorerButtonProps = {
+  title?: string
+  href?: string
+  className?: string
+  icon?: ComponentType
+}
+
+const ExplorerButton = ({ title = '', href = '', icon = LinkIcon, className }: ExplorerButtonProps): ReactElement => (
+  <Tooltip title={title} placement="top">
+    <IconButton
+      className={className}
+      target="_blank"
+      rel="noreferrer"
+      href={href}
+      size="small"
+      sx={{ color: 'inherit' }}
+    >
```

Comment:
There was a `stopPropagation` call onClick here. Is it not needed anymore?

Reply:
Where was it exactly? I can't find it in the change set.

Reply:
It was part of safe-react-components